modules/pool/src/main/PoolList.scala

```diff
@@ -29,6 +29,16 @@ object PoolList {
     PoolConfig(3 ++ 2, Wave(22 seconds, 30 players), Variant.FairySF(strategygames.fairysf.variant.Xiangqi)),
     PoolConfig(3 ++ 2, Wave(22 seconds, 30 players), Variant.FairySF(strategygames.fairysf.variant.Flipello)),
     PoolConfig(3 ++ 2, Wave(22 seconds, 30 players), Variant.FairySF(strategygames.fairysf.variant.Amazons)),
+    PoolConfig(
+      3 ++ 2,
+      Wave(22 seconds, 30 players),
+      Variant.FairySF(strategygames.fairysf.variant.BreakthroughTroyka)
+    ),
+    PoolConfig(
+      3 ++ 2,
+      Wave(22 seconds, 30 players),
+      Variant.FairySF(strategygames.fairysf.variant.MiniBreakthroughTroyka)
+    ),
```

Comment:
I would only add one game here, the default one not both. We only have one variant from each group in the pools.